docs/source/model_design/designing_vivarium_model/interactive_sim/index.rst

```diff
@@ -0,0 +1,116 @@
+====================================
+
+.. contents::
+   :local:
+   :depth: 2
+
+Intro to Interactive Simulations
+--------------------------------
+
+Interactive simulations are a way to step inside of a simulation intsead of 
+just receiving final outputs. This can be very helpful for a number of tasks 
+including V&V, debugging, and reviewing primary outputs. These are described 
+in detail :ref:`below <interactive_tasks>`. 
+
+The interactive sim allows a user to see all of the data available for any simulant. 
+For example, in the main simulation you might record the average BMI in the population 
+by age and sex. In the interactive simulation, you can see each simulant's individual 
+BMI as well as their propensity score for BMI. 
+
+In addition, you can move the simulation forward in time to watch and record how 
+simulants change and experience health events. 
+
+However, we tend to use much smaller populations for the interactive simulation compared to 
+the main simulation. This makes some tasks more difficult to complete interactively. 
+
+.. _interactive_process:
+
+Setting up an Interactive Sim
+-----------------------------
+
+.. todo::
+
+  Add how to: load the interactive sim, change parameters, and take steps 
+
+  Include: - `Vivarium InteractiveContext documentation <https://vivarium.readthedocs.io/en/latest/api_reference/interface/interactive.html?highlight=InteractiveContext#vivarium.interface.interactive.InteractiveContext>`_
+
+
+.. _interactive_tasks:
+
+Example Tasks Using the Interactive Sim
+---------------------------------------
+
+Below are some common tasks that use the interactive sim. Please note that several of 
+these are related to V&V. For more general information and best practices on V&V, see this 
+:ref:`vivarium page <vivarium_best_practices_results_processing>` 
+
+.. todo::
+
+  Add example notebooks to table below. For reference this is the old notebook from the prior V&V page: https://github.com/ihmeuw/vivarium_research_iv_iron/blob/main/validation/maternal/interactive_simulations/Interactive%20simulation%20demo.ipynb 
+
+
+.. list-table:: Common Interactive Simulation Tasks 
+  :widths: 15 15 15
+  :header-rows: 1
+
+  * - Task 
+    - Why is this done interactively? 
+    - Example Notebook 
+  * - V&V for Risks with Many Categories (E.g., LBWSG)
+    - Stratifying simulation outcomes by many categories may be too much of a drain on computation time 
+    - 
+  * - V&V for Continuous Risks
+    - Summary measures such as mean exposure or proportions below a threshold can be simulation outputs. Interactive sims can verify risk exposure standard deviation, look at spread, or check for outliers. 
+    - 
+  * - V&V for Events with Multiple Risk Factors
+    - Strafiying event rates by many risk factors might not be computationally feasible; you can verify risk effects by calculating the event rate at the simulant level.
+    - 
+  * - V&V for Continuous Relative Risks
```

Comment:
```suggestion
  * - V&V for Continuous Risks with Risk Effects
```

Maybe? I don't immediately understand what is meant by "Continuous Relative Risks"